src/engraving/libmscore/scoreorder.cpp

```diff
@@ -175,6 +205,24 @@ bool ScoreOrder::isValid() const
     return !id.isEmpty();
 }
 
+//---------------------------------------------------------
+//   isCustom
+//---------------------------------------------------------
+
+bool ScoreOrder::isCustom() const
+{
+    return id == QString("custom");
+}
+
+//---------------------------------------------------------
+//   getName
+//---------------------------------------------------------
+
+QString ScoreOrder::getName() const
```

Comment:
Better to remove the get prefix. Just name() will be fine

Reply:
This will cause a conflict with the member variable `name` of the same struct.